config/core/resources/channel.yaml

```diff
@@ -74,7 +74,7 @@ spec:
                     description: Spec defines the Spec to use for each channel
                         created. Passed in verbatim to the Channel CRD as Spec
                         section.
-                    type: string
+                    type: object
```

Comment:
It was `object` before, when we had it:

https://github.com/knative/eventing/blob/release-0.15/config/core/resources/channel.yaml#L83-L84


so, good changing back....


was - I think - accidently set to string here:
https://github.com/knative/eventing/commit/31ef0946172e9bafef7235eb7f3b51aa5b0ece69#diff-0231d3c043c857ee3a2e2f93630b183580a12e09d21affb1b1970f483e5c0bd8R77